Fix covergrid preferences: slider release

knockoutMice opened the pull request:
The UI lagged as hell when I tried to change the cover magnification with the slider. Now it only updates the settings / redraws the covers when:
- The user releases the slider
- The new setting is different from the old one

lazka commented:
We have `util.DeferredSignal` for those situations.

How does `mag_scale.connect('value-changed',  util.DeferredSignal(mag_changed, owner=mag_scale))` perform?

Not using value-changed would break other things changing the slider, like the keyboard (or code)

knockoutMice commented:
You're right. I'm looking into DefferedSignal. 

By the way my mouse wheel jumps 5.0... 

Maybe an "Update" or a "Preview" button would be the best solution.

knockoutMice commented:
DeferredSignal behaves the same as the original one and lags heavily.

I introduced a mag_lock boolean to disable updating while sliding with the cursor.
Also fixed the mouse wheel problem (5.0 steps).
